content/components/icon.mdx

```diff
@@ -0,0 +1,216 @@
+---
+title: Icon
+react: https://primer.style/octicons/packages/react
+rails: https://primer.style/octicons/packages/ruby
+figma: https://www.figma.com/file/1ljgTFkT5NKNRfq5hw07JQ/Octicons?node-id=1%3A2&t=Lz5afoq1bbWrD9Yt-1
+description: Our icons are located in a separate library called Octicons, which is available in various implementations including React, Figma, Rails, and Styled System.
+---
+
+import {Box, Text} from '@primer/react'
+
+<img
+  width="960"
+  alt="An image of multiple icons placed in a grid"
+  src="https://user-images.githubusercontent.com/980622/210349372-9392f4b3-4902-4fd3-ab6f-fbd3587f208a.png"
+/>
+
+## Library
+
+At Primer, we utilize the [Octicons](/octicons/) icon library. This package is available in [Ruby](/octicons/packages/ruby), [Jekyll](/octicons/packages/jekyll), [JavaScript](/octicons/packages/javascript), [React](/octicons/packages/react) and [Styled System](/octicons/packages/styled-system).
+
+## Usage
+
+Icons are most appropriate for representing simple, recognizable concepts or actions. When choosing icons for your design system, consider the following guidelines:
+
+1. Where possible use icons to supplement text, rather than replacing it.
+2. Be sure that the meaning of icons is clear when used without accompanying text..
+3. Choose icons that are easily recognizable and accurately depict the intended concept or action.
+4. Maintain consistency in the use of icons across your product or brand to establish a cohesive visual language.
```

Comment:
```suggestion
4. Maintain consistency in the use of icons across your product or brand.
```
I feel the last bit makes it less clear what you mean.

Also does `product or brand` make sense here or would it just be `product`?